config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :merchants, except: [:delete]
+  get "/auth/github", as: "github_login"
```

Comment:
Consider adding the route: get "/users/current", to: "users#current", as: "current_user"